Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -572,11 +577,11 @@ public void E2E_BareProject_FilesToAnalyze()
             // Assert
             result.BuildSucceeded.Should().BeTrue();
 
-            result.AssertExpectedTargetOrdering(
-                TargetConstants.DefaultBuild,
-                TargetConstants.SonarCategoriseProject,
-                TargetConstants.SonarWriteFilesToAnalyze,
-                TargetConstants.SonarWriteProjectData);
+            result.AssertExpectedTargetOrdering(TargetConstants.SonarCategoriseProject,
```

Comment:
You've added `CoreCompile` in this test template. Let's see it asserted here as well.